File: sum_of_numbers_from_0_to_N_7kyu.cs
public class SequenceSum
{
  public static string ShowSequence(int n)
  {    
    int sum_result = 0;
    string result = "";
    
    if (n > 0) {      
      for(int c=0; c<=n; c++){
        result += "+";
        result += c.ToString();
        sum_result += c;
      }
      result += " = " + sum_result.ToString();
      
      result = result.Substring(1);
    } else if(n == 0) {
      result = "0=0";
    } else {
      result = n.ToString() + "<0";
    }
    
    return result;
  }
}

File: sum_of_numbers_from_0_to_N_7kyu.java
public class SequenceSum{

  public static String showSequence(int value){
    int sum_result = 0;
    String result = "";
    
    if (value > 0) {      
      for(int c=0; c<=value; c++){
        result += "+";
        result += Integer.toString(c);
        sum_result += c;
      }
      result += " = " + Integer.toString(sum_result);
      
      result = result.substring(1, result.length());
    } else if(value == 0) {
      result = "0=0";
    } else {
      result = Integer.toString(value) + "<0";
    }
    
    return result;
  }
}
